scripts/local-universe.py

```diff
@@ -62,11 +63,21 @@ def main():
         action='store_true',
         default=False,
         help='Set this to leave CLI resource URLs untouched.')
+    parser.add_argument(
+        '--dcos_version',
+        help='Set this to the version of DC/OS under which the local universe '
+        'will operate. Ensures that only package versions compatible with '
+        'that DC/OS version are included. If unset, the latest version of '
+        'each package will be included.'
+    )
 
     args = parser.parse_args()
 
     package_names = [name for name in args.include.split(',') if name != '']
 
+    dcos_version = distutils.version.LooseVersion(args.dcos_version) \
+        if args.dcos_version is not None else None
```

Comment:
Okay but if you want `... if args.dcos_version else None`. In Python `None` and `''` (empty string) are `False` in an `if` statement.